packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -415,6 +415,10 @@ const imageNodeType = ({
       },
       maxHeight: {
         type: GraphQLInt,
+        description: stripIndent`
+        If set, the generated image is a maximum of this height, cropping if necessary. 
+        If the image layout is "constrained" then the image will be limited to this height. 
```

Comment:
Is this true? It's limited to maxHeight for constrained unless the original image height is less than that.